includes/admin/admin-actions.php

```diff
@@ -1509,6 +1509,44 @@ function give_admin_quick_js() {
 
 add_action( 'admin_head', 'give_admin_quick_js' );
 
+/**
+ * Admin Footer JS
+ *
+ * @since 2.6.0
+ */
+function give_admin_footer_quick_js() {
```

Comment:
Please update the name of this function so it is more descriptive to add upsell JS.